R/sjTabItemAnalysis.R

```diff
@@ -92,24 +92,22 @@
 #'             }
 #' 
 #' @examples
-#' # Data from the EUROFAMCARE sample dataset
-#' library(sjmisc)
-#' data(efc)
-#' 
-#' # retrieve variable and value labels
+#' Data from the EUROFAMCARE sample dataset
+#' library(sjmisc)data(efc)
+#' retrieve variable and value labels
 #' varlabs <- get_label(efc)
 #' 
 #' # recveive first item of COPE-index scale
 #' start <- which(colnames(efc) == "c82cop1")
 #' # recveive last item of COPE-index scale
 #' end <- which(colnames(efc) == "c90cop9")
-#'  
+#' 
 #' # create data frame with COPE-index scale
 #' mydf <- data.frame(efc[, c(start:end)])
 #' colnames(mydf) <- varlabs[c(start:end)]
-#' 
-#' \dontrun{
-#' sjt.itemanalysis(mydf)
+#' #' #'
+#' #' #' \dontrun{
```

Comment:
You could write tests with package *testthat* ;-) But, copy/paste from the docs to console works, you only need to uncomment when doing R CMD check. However, opening web browsers in examples is not good practice according CRAN guidelines, hence I added the dont-run-tag to all HTML-functions.